src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx

```diff
@@ -0,0 +1,144 @@
+import { getTranslation, ResourceId } from '@esss-swap/duo-localisation';
+import {
+  createProposalStatusValidationSchema,
+  updateProposalStatusValidationSchema,
+} from '@esss-swap/duo-validation/lib/ProposalStatuses';
+import Button from '@material-ui/core/Button';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import Typography from '@material-ui/core/Typography';
+import { Field, Form, Formik } from 'formik';
+import { TextField } from 'formik-material-ui';
+import { useSnackbar } from 'notistack';
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+
+import UOLoader from 'components/common/UOLoader';
+import { ProposalStatus } from 'generated/sdk';
+import { useDataApi } from 'hooks/common/useDataApi';
+
+const useStyles = makeStyles(theme => ({
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+type CreateUpdateProposalStatusProps = {
+  close: (proposalStatusAdded: ProposalStatus | null) => void;
+  proposalStatus: ProposalStatus | null;
+};
+
+const CreateUpdateProposalStatus: React.FC<CreateUpdateProposalStatusProps> = ({
+  close,
+  proposalStatus,
+}) => {
+  const classes = useStyles();
+  const api = useDataApi();
```

Comment:
You could use useDataApiWithFeedback here and then remove error displaying in lines :57-:62

Reply:
Thanks @jekabs-karklins I saw it and it is coming in the next PR that I am currently working on https://github.com/UserOfficeProject/user-office-frontend/blob/d558d2f36ca6412b2fcfc9da98da3c89d1bf8c8d/src/components/settings/proposalStatus/CreateUpdateProposalStatus.tsx#L33